src/main/resources/reports/residential-usage.json

```diff
@@ -80,7 +101,10 @@
                 {"name": "CELL_CONVERTION_TO_ROOM", "display": "Cell conversion to room"},
                 {"name": "ROOM_CONVERTION_TO_CELL", "display": "Room conversion to cell"},
                 {"name": "SIGNED_OP_CAP", "display": "Signed operational capacity"},
-                {"name": "RESI_SERVICE_ACTIVATION", "display": "Residential service activation"}
+                {"name": "RESI_SERVICE_ACTIVATION", "display": "Residential service activation"},
+                {"name": "APPROVAL_PROCESS_ACTIVATION", "display": "Certification approval activation"},
+                {"name": "SYNC", "display": "NOMIS sync"},
```

Comment:
Decided to allow users to see the NOMIS syncs if they want, but not selected by default